packages/world-modules/mud.config.ts

```diff
@@ -190,7 +190,7 @@ export default mudConfig({
       },
       tableIdArgument: true,
     },
-    TokenURI: {
+    ERC721TokenURI: {
```

Comment:
Maybe ERC721 and ERC1155 could share tables like `TokenURI` or `OperatorApproval` that have the same schema? 

Just like `Balances`, in the `modules/tokens/tables` directory: https://github.com/MerkleBoy/mud/blob/merkleboy/erc1155-puppet-module/packages/world-modules/mud.config.ts#L121